scripts/eslint.py

```diff
@@ -49,20 +49,25 @@ def run_eslint():
     )
 
     print(result.stdout)
-    last_line = result.stdout.strip().splitlines()[-1] if result.stdout.strip().splitlines() else ""
-    regex = r'^\d+'
-    try:
-        num_violations = int(re.search(regex, last_line).group(0)) if last_line else 0
-        # Fail if number of violations is greater than the limit
-        if num_violations > violations_limit:
-            fail_quality(
-                "FAILURE: Too many eslint violations ({count}).\nThe limit is {violations_limit}.".format(count=num_violations, violations_limit=violations_limit))
-        else:
-            print(f"successfully run eslint with '{num_violations}' violations")
+    if result.returncode == 0:
+        print("No eslint violations found.")
```

Comment:
```suggestion
        fail_quality("No eslint violations found. This is unexpected... are you sure eslint is running correctly?")
```
Normally we would be happy to see zero violations, but in the case of eslint for edx-platform, zero violations would mostly likely indicate that eslint is not running correctly. So, let's fail the check here. If we eventually remove all the violations, we can change this back to a pass.

Reply:
@kdmccormick this is making sense to me. i am merging this commit suggestion.